test/chpldoc/deprecatedSymbols.doc.chpl

```diff
@@ -0,0 +1,18 @@
+// Checks basic deprecated declarations of variables
+deprecated var x: int;
+deprecated "y is deprecated, use z instead" var y: int;
+var z: int;
+
+// Checks interaction with documentation
+/* There was documentation of this symbol */
+deprecated var a: string;
+/* This symbol also was documented */
+deprecated "b is deprecated, use c instead" var b: string;
+var c: string;
+
+// Checks interaction when documentation mentions deprecation in some form
+/* This symbol is deprecated */
```

Comment:
Ideally wouldn't we also look for a warning tag? Or maybe that's not possible/feasible? This case is odd because while it mentions deprecation it doesn't seem to be `..warning::` in the generated docs.

Reply:
The user could be using chpldoc with `--text-only` mode, so wouldn't want restructured text commands.  Or they might prefer to use `.. note::` blocks instead.  Basically, I'm trying to avoid expecting a specific style

Reply:
I didn't consider that. That makes sense, thanks!